### PR TITLE
Add version to all telemetry events, and properly trigger start-up telemetry

### PR DESCRIPTION
**Pull request by digeff**
We report the node version and Versions.DebugAdapter as a common telemetry property
We also trigger ChromeDebugSession.FinishedStartingUpEventName to properly mark the end of startup and trigger the end of the report-start-up-timings telemetry event

### CODE REVIEW

File: src/nodeDebugAdapter.ts [hunk 1]
```diff
@@ -457,23 +457,26 @@ export class NodeDebugAdapter extends ChromeDebugAdapter {
...
+            await this.continue(/*internal=*/true);
         } else if (this._entryPauseEvent) {
-            this.onPaused(this._entryPauseEvent);
+            await this.onPaused(this._entryPauseEvent);
         }
 
-        return super.configurationDone();
+        await super.configurationDone();
+
+        this.events.emit(ChromeDebugSession.FinishedStartingUpEventName);
```
Comment (roblourens) [inline src/nodeDebugAdapter.ts]: Does this have to go after the super call? Would rather leave the super call as the last line unless there's a reason for it.

File: src/nodeDebugAdapter.ts [hunk 2]
```diff
@@ -457,23 +457,26 @@ export class NodeDebugAdapter extends ChromeDebugAdapter {
         }
     }
 
-    public configurationDone(): Promise<void> {
+    public async configurationDone(): Promise<void> {
         if (!this.chrome) {
             // It's possible to get this request after we've detached, see #21973
-            return super.configurationDone();
+            await super.configurationDone();
+            return;
```
Comment (roblourens) [inline src/nodeDebugAdapter.ts]: This is equivalent to `return super.configurationDone()`, you could leave it that way for simplicity

Comment (digeff) [reply src/nodeDebugAdapter.ts]: Changed

File: src/nodeDebugAdapter.ts (hunk 1 above)
Comment (digeff) [reply src/nodeDebugAdapter.ts]: Changed

**Comment by roblourens** [comment]
Lots of CI failures?

**Pull request merged**